js/script.js

```diff
@@ -37,16 +37,68 @@ const deleteTaskHandler = (event) => {
     taskList = taskList.filter((task) => task.id!==taskId);
 };
 
+const updateTaskEditHandler = (task, $taskDetails, $inputField, $editButton, $updateButton, $cancelButton) => {
+    task.title = $inputField.value;
+    
+    taskList.map((t) => {
```

Comment:
It's not recommended to use meaningless variable name rather we can use `todo` or `singleTask` insted of `t`